src/methods/abstract.js

```diff
@@ -54,11 +56,23 @@ export function SplitMatch(realm: Realm, S: string, q: number, R: string): false
 }
 
 // ECMA262 7.2.1
-export function RequireObjectCoercible<T: Value>(realm: Realm, arg: T): T {
-  if (HasSomeCompatibleType(arg, NullValue, UndefinedValue)) {
+export function RequireObjectCoercible(
```

Comment:
This typing seems less specific than the old typing, is it possible to leave it templated and have flow enforce that?